Explicitly mention aws repo extra url in documentation (#277)
huggingface/optimum-neuron: docs/source/installation.mdx

# Installation

Our recommendation is to use the [Hugging Face Neuron Deep Learning AMI](https://aws.amazon.com/marketplace/pp/prodview-gr3e6yiscria2) (DLAMI). The DLAMI comes with all required libraries pre-packaged for you, including the Optimum Neuron, Neuron Drivers, Transformers, Datasets, and Accelerate.

But it can also be installed using `pip` as described below.

Before installing `optimum-neuron` make sure that you have installed the Neuron driver and tools, check out [more detailed guide here](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/setup/torch-neuronx.html#setup-torch-neuronx).

> ## Adding AWS neuron SDK pip packages repo URL

Reviewer: ```suggestion
## Adding pip packages URL

Pointing to the AWS Neuron repository:
```
